packages/graphql/src/classes/Node.ts

```diff
@@ -176,6 +160,14 @@ class Node {
     get labels(): string[] {
         return this.nodeDirective?.getLabels(this.name) || [this.name];
     }
+
+    getPlural(camelcase: boolean): string {
```

Comment:
As previously discussed, the parameter can be removed by fixing the inconsistencies between queries and mutations in automatic casing. However, that would be a breaking change requiring a major version update